### ISSUE TITLE
Key resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut_bn/BatchNorm/beta not found in checkpoint

### ISSUE THREAD

**Issue by darchonyzx** [auroua/InsightFace_TF#61]
Key resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut_bn/BatchNorm/beta not found in checkpoint

tensorflow == 1.6.0
tensorlayer== 1.7.0

**Comment by zjz5250** [comment]
l  have  the problem

**Comment by cychen3** [comment]
do you solve this problem? i have the problem "FailedPreconditionError (see above for traceback): Attempting to use uninitialized value resnet_v1_50/block4/unit_3/bottleneck_v1/conv2_bn/BatchNorm/beta"

**Comment by lovejing0306** [comment]
you need use L_Resnet_E_IR_fix_issue9 

> do you solve this problem? i have the problem "FailedPreconditionError (see above for traceback): Attempting to use uninitialized value resnet_v1_50/block4/unit_3/bottleneck_v1/conv2_bn/BatchNorm/beta"



> l have the problem



> Key resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut_bn/BatchNorm/beta not found in checkpoint
> 
> tensorflow == 1.6.0
> tensorlayer== 1.7.0
